scripts/zones/Arrapago_Reef/Zone.lua

```diff
@@ -8,6 +8,18 @@ zoneObject.onInitialize = function(zone)
     zone:registerCuboidTriggerArea(1, -462, -4, -420, -455, -1, -392) -- approach the Cutter
 end
 
+zoneObject.onPlayerDeath = function(player)
+    local ID = require('scripts.zones.Arrapago_Reef.IDs')
```

Comment:
We store the ID tables in a global
```suggestion
    local ID = zones[xi.zone.ARRAPAGO_REEF]
```